src/main/java/Main.java

```diff
@@ -1,9 +1,36 @@
-/**
- * Created by iyasuwatts on 10/17/17.
- */
+
+import java.util.Scanner;
 public class Main {
 
     public static void main(String[] args ){
 
+        Scanner input = new Scanner(System.in);
+        System.out.println("What is your name?");
+        String name = input.nextLine();
+        if (isBob(name)){
+            System.out.println("Heya Bob, how're the kids?");
+        }
+        else if (isAlice(name)){
+            System.out.println("Alice!  It's been a while!  How's the Tennis Elbow?");
+        }
+        else{
+            System.out.println("You're not Bob or Alice!  Get the hell out of here!");
+        }
+
+
+    }
+
+    public static boolean isBob (String name){
+        if (name.equals("Bob") || name.equals("bob")){
+            return true;
+        }
+            return false;
+    }
+
+    public static boolean isAlice (String name){
```

Comment:
Thumbs up for the `SRP` 👍 
Thumbs down for the code redundancy...

Notice the similarities between `isBob` and `isAlice`.
When you see patterns of redundancy like this, it means it can be _abstracted_.